lib/src/model/tournament/tournament.dart

```diff
@@ -0,0 +1,89 @@
+import 'package:deep_pick/deep_pick.dart';
+import 'package:fast_immutable_collections/fast_immutable_collections.dart';
+import 'package:freezed_annotation/freezed_annotation.dart';
+import 'package:lichess_mobile/src/model/common/chess.dart';
+import 'package:lichess_mobile/src/model/common/id.dart';
+import 'package:lichess_mobile/src/model/common/perf.dart';
+import 'package:lichess_mobile/src/model/common/time_increment.dart';
+import 'package:lichess_mobile/src/model/user/user.dart';
+import 'package:lichess_mobile/src/utils/json.dart';
+
+part 'tournament.freezed.dart';
+
+typedef TournamentSchedule = ({String freq, String speed});
```

Comment:
Is the speed specific to tournament? If not we want to use the `Speed` enum.

(If specific to tournament perhaps an enum is better also).

Reply:
Ah yes this seems to be specific to tournaments I think: https://github.com/lichess-org/lila/blob/ede805b16b1e9738c908e177562c742cea5b8865/modules/tournament/src/main/Schedule.scala#L86

But we don't actually use it right now, so let's just remove it? I added an enum for the `freq` field though (see https://github.com/lichess-org/lila/blob/ede805b16b1e9738c908e177562c742cea5b8865/modules/tournament/src/main/Schedule.scala#L60)